Python/Camscanner/README.md

```diff
@@ -0,0 +1,15 @@
+# Generate Scanned Image
```

Comment:
Hey @SmritiAgrawal04 Can we please have a more detailed README 😅

Reply:
Done! Can you please check if this looks good now?